org.freedesktop.Sdk.Extension.mono5.json

```diff
@@ -8,43 +9,26 @@
     "appstream-compose": false,
     "modules": [
         {
-            "name": "python-2.7",
-            "sources": [
-                {
-                    "type": "archive",
-                    "url": "https://www.python.org/ftp/python/2.7.17/Python-2.7.17.tar.xz",
-                    "sha256": "4d43f033cdbd0aa7b7023c81b0e986fd11e653b5248dac9144d508f11812ba41"
-                }
-            ],
-            "config-opts": [
-                "--enable-shared",
-                "--with-ensurepip=yes",
-                "--with-system-expat",
-                "--with-system-ffi",
-                "--enable-loadable-sqlite-extensions",
-                "--with-dbmliborder=gdbm",
-                "--enable-unicode=ucs4"
-            ],
-            "post-install": [
-                "chmod 644 /app/lib/libpython2.7.so.1.0",
-                "ln -s /app/bin/python2 /app/bin/python"
-            ],
-            "cleanup": ["*"]
+            "name": "python",
```

Comment:
You want to keep `"cleanup": ["*"]`.

Reply:
This module only creates a single symlink. What will be cleaned up here?

Reply:
It will remove the symlink. Is it needed for mono to run?

Reply:
It is needed for the mono build step. I don't know if some of the scripts which explicitly use `python` are needed at runtime as well. I never used it. Just wanted to see https://github.com/flathub/com.github.PintaProject.Pinta/issues/7 fixed...

Reply:
If it's needed only for mono build step and is not a runtime dependency, it should not be here.

Reply:
I am asking the upstream developers. Where should it be if it's not needed?

Reply:
Build time dependencies are build time. Cleanup will simply remove it after all modules are built.